Scancode failed for maven/io.vertx:vertx-config/3.4.0.Beta1

```
2017-09-13 07:30:55,705 [ERROR] celery.app.trace: Task selinon.SelinonTaskEnvelope[15aa23f8-aae2-4318-8fbe-ec79898ee30b] raised unexpected: FatalTaskError('Error (137) during running command [\'/opt/scancode-toolkit/scancode\', \'--license\', \'--license-score\', \'20\', \'--only-findings\', \'--processes\', \'2\', \'--quiet\', \'--strip-root\', \'--timeout\', \'120\', \'/var/lib/f8a_worker/worker_data/maven/io.vertx:vertx-config/3.4.0.Beta1/extracted_jar\', \'--ignore\', \'*.pyc\', \'--ignore\', \'*.so\', \'--ignore\', \'*.dll\', \'--ignore\', \'*.rar\', \'--ignore\', \'*.jar\', \'--ignore\', \'*.zip\', \'--ignore\', \'*.tar\', \'--ignore\', \'*.tar.gz\', \'--ignore\', \'*.tar.xz\']: \'/opt/scancode-toolkit/scancode: line 114:   852 Killed                  $SCANCODE_ROOT_DIR/bin/scancode "$@"\\n\'',)
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/celery/app/trace.py", line 367, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/celery/app/trace.py", line 622, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/selinon/selinonTaskEnvelope.py", line 170, in run
    raise self.retry(max_retries=0, exc=exc)
  File "/usr/lib/python3.4/site-packages/celery/app/task.py", line 668, in retry
    raise_with_context(exc)
  File "/usr/lib/python3.4/site-packages/selinon/selinonTaskEnvelope.py", line 115, in run
    result = task.run(node_args)
  File "/usr/lib/python3.4/site-packages/f8a_worker/base.py", line 38, in run
    result = self.execute(node_args)
  File "/usr/lib/python3.4/site-packages/f8a_worker/workers/license.py", line 108, in execute
    result_data = self.run_scancode(cache_path)
  File "/usr/lib/python3.4/site-packages/f8a_worker/workers/license.py", line 80, in run_scancode
    raise FatalTaskError("Error (%s) during running command %s: %r" % (str(status), command, error))
selinon.errors.FatalTaskError: Error (137) during running command ['/opt/scancode-toolkit/scancode', '--license', '--license-score', '20', '--only-findings', '--processes', '2', '--quiet', '--strip-root', '--timeout', '120', '/var/lib/f8a_worker/worker_data/maven/io.vertx:vertx-config/3.4.0.Beta1/extracted_jar', '--ignore', '*.pyc', '--ignore', '*.so', '--ignore', '*.dll', '--ignore', '*.rar', '--ignore', '*.jar', '--ignore', '*.zip', '--ignore', '*.tar', '--ignore', '*.tar.gz', '--ignore', '*.tar.xz']: '/opt/scancode-toolkit/scancode: line 114:   852 Killed                  $SCANCODE_ROOT_DIR/bin/scancode "$@"\n'
```

I see similar error multiple times, e.g. for maven/com.networknt:mask/1.1.0:

```
2017-09-13 07:31:07,549 [ERROR] celery.app.trace: Task selinon.SelinonTaskEnvelope[e08a3cc6-85a4-4040-bc7b-1413ffc6b75b] raised unexpected: FatalTaskError('Error (137) during running command [\'/opt/scancode-toolkit/scancode\', \'--license\', \'--license-score\', \'20\', \'--only-findings\', \'--processes\', \'2\', \'--quiet\', \'--strip-root\', \'--timeout\', \'120\', \'/var/lib/f8a_worker/worker_data/maven/com.networknt:mask/1.1.0/extracted_jar\', \'--ignore\', \'*.pyc\', \'--ignore\', \'*.so\', \'--ignore\', \'*.dll\', \'--ignore\', \'*.rar\', \'--ignore\', \'*.jar\', \'--ignore\', \'*.zip\', \'--ignore\', \'*.tar\', \'--ignore\', \'*.tar.gz\', \'--ignore\', \'*.tar.xz\']: \'/opt/scancode-toolkit/scancode: line 114:   896 Killed                  $SCANCODE_ROOT_DIR/bin/scancode "$@"\\n\'',)
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/celery/app/trace.py", line 367, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/celery/app/trace.py", line 622, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/selinon/selinonTaskEnvelope.py", line 170, in run
    raise self.retry(max_retries=0, exc=exc)
  File "/usr/lib/python3.4/site-packages/celery/app/task.py", line 668, in retry
    raise_with_context(exc)
  File "/usr/lib/python3.4/site-packages/selinon/selinonTaskEnvelope.py", line 115, in run
    result = task.run(node_args)
  File "/usr/lib/python3.4/site-packages/f8a_worker/base.py", line 38, in run
    result = self.execute(node_args)
  File "/usr/lib/python3.4/site-packages/f8a_worker/workers/license.py", line 108, in execute
    result_data = self.run_scancode(cache_path)
  File "/usr/lib/python3.4/site-packages/f8a_worker/workers/license.py", line 80, in run_scancode
    raise FatalTaskError("Error (%s) during running command %s: %r" % (str(status), command, error))
selinon.errors.FatalTaskError: Error (137) during running command ['/opt/scancode-toolkit/scancode', '--license', '--license-score', '20', '--only-findings', '--processes', '2', '--quiet', '--strip-root', '--timeout', '120', '/var/lib/f8a_worker/worker_data/maven/com.networknt:mask/1.1.0/extracted_jar', '--ignore', '*.pyc', '--ignore', '*.so', '--ignore', '*.dll', '--ignore', '*.rar', '--ignore', '*.jar', '--ignore', '*.zip', '--ignore', '*.tar', '--ignore', '*.tar.gz', '--ignore', '*.tar.xz']: '/opt/scancode-toolkit/scancode: line 114:   896 Killed                  $SCANCODE_ROOT_DIR/bin/scancode "$@"\n'
```

--- a/openshift/template.yaml
+++ b/openshift/template.yaml
@@ -242,7 +242,7 @@ objects:
               memory: "512Mi"
               cpu: "300m"
             limits:
-              memory: "1200Mi"
+              memory: "1536Mi"
               cpu: "500m"
         restartPolicy: Always
     test: false

